### PR TITLE
py: implement foreach_chunk method for streaming HTTP output

### PR DESCRIPTION
**Pull request by abhizer**
Is this a user-visible change (yes/no): yes

### CODE REVIEW

File: python/feldera/_callback_runner.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+class CallbackRunner(Thread):
+    def __init__(
+            self,
+            client: FelderaClient,
+            pipeline_name: str,
+            view_name: str,
+            callback: Callable[[pd.DataFrame, int], None],
+            queue: Queue,
+    ):
```
Comment (ryzhyk) [inline python/feldera/_callback_runner.py:19]: This code is almost identical to `output_handler.rs`.  I understand that that code assembles the result in a single dataframe, while here we invoke the callback for each batch.  But I would expect the former to be implemented on top of the latter, so we shouldn't need two near-identical implementations.

File: python/feldera/_callback_runner.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+                for chunk in gen_obj:
+                    chunk: dict = chunk
+                    data: list[dict] = chunk.get("json_data")
+                    seq_no: int = chunk.get("sequence_number")
+
+                    if data is not None:
+                        self.callback(dataframe_from_response([data]), seq_no)
+
+                    try:
```
Comment (ryzhyk) [inline python/feldera/_callback_runner.py:49]: Can you please add comments explaining what's going on here?

File: python/feldera/_callback_runner.py (hunk 1 above)
Comment (abhizer) [reply python/feldera/_callback_runner.py:19]: Yes. I am working on finding an elegant way to merge the two.

**Comment by ryzhyk** [comment]
I also realized that this still doesn't break input into lines in `listen_to_pipeline`.

**Comment by ryzhyk** [comment]
Ran into another issue: `SQLContext.start` doesn't process `self.views_tx` and `self.http_input_buffer`, so `foreach_chunk` doesn't work when using it.

**Pull request merged**